packages/@lwc/synthetic-shadow/src/faux-shadow/events.ts

```diff
@@ -338,24 +332,17 @@ function isValidEventForCustomElement(event: Event): boolean {
 }
 
 export function addCustomElementEventListener(
-    elm: HTMLElement,
+    elm: Element,
     type: string,
     listener: EventListener,
-    options?: boolean | AddEventListenerOptions
+    _options?: boolean | AddEventListenerOptions
```

Comment:
Was this change necessary or does the underscore imply something? Is it because we're not using this argument but we need it for typescript?

Reply:
yes, it is exactly that. we have that rule in the ts config.